build.gradle.kts

```diff
@@ -46,62 +47,53 @@ kotlin {
         }
     }
     jvm {
-        compilations.all {
-            kotlinOptions.jvmTarget = jvmTarget.toString()
-        }
-        withJava()
         testRuns["test"].executionTask.configure {
             useJUnitPlatform()
         }
-    }
-    jvmToolchain {
-        languageVersion = JavaLanguageVersion.of(javaVersion)
+        @OptIn(ExperimentalKotlinGradlePluginApi::class)
+        compilerOptions {
+            // note lang toolchain cannot be used here
+            // because gradle no longer supports running on java 8
+            jvmTarget = jvm
```

Comment:
[This is not true?](https://docs.gradle.org/current/userguide/compatibility.html#kotlin)

I'm not opposed to bumping the JVM target version, but it should be 11 (LTS) instead of 10, and preferably for a good reason.

Reply:
right, my bad, I was getting some weird error earlier. It seems 1.8 works after all. I meant to update to 11 but was getting an error. I'll push the fix.